src/Components/Pages/dashboard.module.css

```diff
@@ -0,0 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(45, 45, 105);
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  background: rgb(29, 29, 70);
+  justify-content: space-between;
+  min-height: 4rem;
+  padding: 0.5rem;
+  color: white;
+}
+
+.brand {
+  margin-left: 40px;
+}
+
+.stocklist {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  color: white;
+  font-size: 17px;
+}
```

Comment:
Your app works great! However, the styling of the list on your _Homepage_ does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that:
- [ ] two cards per row are displayed for mobile screen, as seen in the template design.
- [ ] the cards of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (or slightly alternating color shade) background color(s)**.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/217349204-c9e84280-c30f-4248-9841-f6480ae697f0.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png">

Reply:
Behance blue theme dispatched